src/storable.rs

```diff
@@ -344,6 +344,27 @@ impl Storable for u8 {
     };
 }
 
+impl Storable for bool {
+    fn to_bytes(&self) -> Cow<[u8]> {
+        let num: u8 = if *self { 1 } else { 0 };
+        Cow::Owned(num.to_be_bytes().to_vec())
+    }
+
+    fn from_bytes(bytes: Cow<[u8]>) -> Self {
+        let num = u8::from_be_bytes(bytes.as_ref().try_into().unwrap());
+        match num {
+            0 => false,
+            1 => true,
+            _  => panic!("Invalid bool encoding: expected 0 or 1, found {}", num)
+        }
```

Comment:
This can be made more efficient by avoiding the creation of a `u8` altogether:

```suggestion
        assert_eq!(bytes.len(), 1);
        match bytes[0] {
            0 => false,
            1 => true,
            other => panic!("Invalid bool encoding: expected 0 or 1, found {}", other),
        }
```